src/algebra/group/prod.lean

```diff
@@ -0,0 +1,232 @@
+/-
+Copyright (c) 2020 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Simon Hudon, Patrick Massot, Yury Kudryashov
+-/
+
+import algebra.group.hom
+
+variables {A : Type*} {B : Type*} {G : Type*} {H : Type*} {M : Type*} {N : Type*} {P : Type*}
+
+namespace prod
+
+-- TODO (Lean 3.7.0) : make `to_additive` aware of `protected`, and use it for instances
+-- in this file
+
+instance [has_add A] [has_add B] : has_add (A × B) := ⟨λ p q, ⟨p.1 + q.1, p.2 + q.2⟩⟩
```

Comment:
Why do you need to explicitly state all the additive versions? Can't `to_additive` do that for you? Is this related to the comment above?

Reply:
Yes, `to_additive` would've created non-`protected` `prod.has_add`.

Reply:
Fixed now.